File: TuristickaAgencija.WebAPI/Database/Korisnici.cs
using System;
using System.Collections.Generic;

// Code scaffolded by EF Core assumes nullable reference types (NRTs) are not used or disabled.
// If you have enabled NRTs for your project, then un-comment the following line:
// #nullable disable

namespace TuristickaAgencija.WebAPI.Database
{
    public partial class Korisnici
    {
        public Korisnici()
        {
            Komentar = new HashSet<Komentar>();
            KorisniciUloge = new HashSet<KorisniciUloge>();
            Obavijesti = new HashSet<Obavijesti>();
            Ocjene = new HashSet<Ocjene>();
            Rezervacija = new HashSet<Rezervacija>();
            ListaZelja = new HashSet<ListaZelja>();
        }

        public int Id { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string Email { get; set; }
        public string Telefon { get; set; }
        public string KorisnickoIme { get; set; }
        public string LozinkaHash { get; set; }
        public string LozinkaSalt { get; set; }
        public bool? Status { get; set; }

        public virtual ICollection<Komentar> Komentar { get; set; }
        public virtual ICollection<KorisniciUloge> KorisniciUloge { get; set; }
        public virtual ICollection<Obavijesti> Obavijesti { get; set; }
        public virtual ICollection<Ocjene> Ocjene { get; set; }
        public virtual ICollection<Rezervacija> Rezervacija { get; set; }
        public virtual ICollection<ListaZelja> ListaZelja { get; set; }
    }
}

File: UI/turisticka_agencija_admin/lib/models/Korisnici.dart
import 'KorisniciUloge.dart';
import 'package:json_annotation/json_annotation.dart';

part 'Korisnici.g.dart';

@JsonSerializable()
class Korisnici {
    String email;
    int id;
    String ime;
    List<KorisniciUloge>? korisniciUloge;
    String korisnickoIme;
    String lozinkaHash;
    String lozinkaSalt;
    String prezime;
    bool status;
    String telefon;

    Korisnici({required this.email, required this.id,required this.ime,required this.korisniciUloge,required this.korisnickoIme,required this.lozinkaHash,required this.lozinkaSalt,required this.prezime,required this.status,required this.telefon});

    factory Korisnici.fromJson(Map<String, dynamic> json) => _$KorisniciFromJson(json);

    Map<String, dynamic> toJson() => _$KorisniciToJson(this);
}